proxy/proxy.go

```diff
@@ -235,13 +240,19 @@ func newCattleProxies(config *Config) (*proxyProtocolConverter, *cattleWSProxy)
 }
 
 type proxyProtocolConverter struct {
-	reverseProxy *httputil.ReverseProxy
-	httpsPorts   map[int]bool
+	reverseProxy        *httputil.ReverseProxy
+	httpsPorts          map[int]bool
+	apiFilterProxy      *apifilterproxy.APIFiltersHandler
+	apiFilterProxyReady bool
 }
 
 func (h *proxyProtocolConverter) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 	proxyprotocol.AddHeaders(req, h.httpsPorts)
-	h.reverseProxy.ServeHTTP(rw, req)
+	if h.apiFilterProxyReady {
```

Comment:
Yes, I was trying to play safe while integrating this to fallback on your existing cattleproxy. But I dont think I need it anymore, will always call my proxy to handle.